File: year2020/month12/day1208/ms_1208.cs
using System;

namespace algorithm_117
{
    class ms_1208
    {
        static string solution(int n)
        {
            string answer = "";

            for(int i = 0; i < n/2; i++)
            {
                answer += "수박";
            }

            if(answer.Length != n)
            {
                answer += "수";
            }

            return answer;
        }

        static void Main(string[] args)
        {
            int n = Convert.ToInt32(Console.ReadLine());

            if(1 <= n && n <= 10000)
            {
                Console.WriteLine(solution(n));
            }
        }
    }
}

File: year2020/month12/day1208/ms_1208.cpp
#include <iostream>
#include <string>

using namespace std;

string solution(int n);

int main() {
	int n;

	cin >> n;

	if (1 <= n && n <= 10000) {
		cout << solution(n) << endl;
	}

	return 0;
}

string solution(int n) {
	string answer = "";
	int len = 0;

	for (int i = 0; i < n / 2; i++) {
		answer += "¼ö¹Ú";
		len = (i + 1) * 2;
	}
	
	if (len != n) {
		answer += "¼ö";
	}

	return answer;
}